### PR TITLE
Feature queries sheet 61-65

### PR DESCRIPTION
**Pull request by esirK**
## Description
Updates queries for sheet 61-65

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Screenshots
# 61
Screenshot 2021-03-08 at 13 29 12

# 62 
Screenshot 2021-03-09 at 04 15 41

#  63
Screenshot 2021-03-09 at 04 21 02

# 64
Screenshot 2021-03-09 at 04 18 02

# 65

Screenshot 2021-03-08 at 15 28 58

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by kilemensi — COMMENTED** [review]
👍🏾 @esirK ... `61` and `65` are good to go.

In `62`, `63` an `64`, [it should be](https://docs.google.com/spreadsheets/d/1FV6pUcW3hEade4y17ENGAP5Dp4RkNnDW/edit#gid=1308543941&range=K20:K22) **MAJOR_TOPICS** and not _TOPICS_.

File: reports/report_builder.py [hunk 1]
```diff
@@ -2025,128 +2025,102 @@ def ws_60(self, ws):
...
+        rows = model.objects\
+                .values('sex', 'is_quoted')\
+                .filter(**{model.sheet_name() + "__country__in": self.country_list}) \
+                .annotate(n=Count('id'))
 
-            rows = self.apply_weights(rows, model.sheet_db_table(), "Internet")
-            counts = {(row['sex'], row['is_quoted']): row['n'] for row in rows}
+        rows = self.apply_weights(rows, model.sheet_db_table(), "Internet")
+        for row in rows:
+            counts[row['sex'], row['is_quoted']] += row['n']
```
Comment (kilemensi) [inline reports/report_builder.py]: Just out of curiosity, why are we updating `counts` here differently from the usual `counts.update`?

**Review by kilemensi — APPROVED** [review]
🚀

File: reports/report_builder.py (hunk 1 above)
Comment (esirK) [reply reports/report_builder.py]: Mainly just for readability. I did this while trying to understand what was happening. I'll revert it back for consistency.

Comment (esirK) [reply reports/report_builder.py]: @kilemensi  Also, I had noticed some differences between the 2 approaches. 
For example with the 
`counts = {(row['sex'], row['is_quoted']): row['n'] for row in rows}` approach, the sum of the counts doesn't add up to the `model.objects` count. 

Here is an example on sheet 62
```
counts = Counter()
counts1 = Counter()

for row in rows:
    counts[TOPIC_GROUPS[row['topic']], row['equality_rights']] += row['n']
counts1 = {(TOPIC_GROUPS[row['topic']], row['equality_rights']): row['n'] for row in rows}

counts1 == counts # returns false
sum([c for c in counts.values()]) == 4703
sum([c for c in counts1.values()]) == 918
```
The total value in the database is `4703`.
Unless I'm missing something, we might have been returning incorrect values.

Comment (kilemensi) [reply reports/report_builder.py]: Does calling `count.update` make any difference @esirK?  i.e. `counts.update({(r["topic"], r["equality_rights"]): r["n"] for r in rows})`

Comment (esirK) [reply reports/report_builder.py]: No, it doesn't.

Comment (esirK) [reply reports/report_builder.py]: I've noticed this only happens when we are using dict comprehension for the `row`. Looping through the rows normally and doing the `count.update` works fine.

**Pull request merged**